wpilib/wpilib/speedcontrollergroup.py

```diff
@@ -7,15 +7,33 @@
 # the project.
 # ----------------------------------------------------------------------------
 from .interfaces.speedcontroller import SpeedController
+from .sendablebase import SendableBase
 
 __all__ = ["SpeedControllerGroup"]
 
 
-class SpeedControllerGroup(SpeedController):
+class SpeedControllerGroup(SendableBase, SpeedController):
+    """Allows multiple :class:`.SpeedController` objects to be linked together"""
+
+    instances = 0
+
     def __init__(self, *args):
+        """Create a new SpeedControllerGroup with the provided SpeedControllers.
+
+        :param args: SpeedControllers to add
+        :type args: :class:`.SpeedController`
+        """
+        SendableBase.__init__(self)
+        SpeedController.__init__(self)
+
+        for speedController in args:
+            self.addChild(speedController)
         self.speedControllers = args
         self.isInverted = False
 
+        self.instances += 1
```

Comment:
I think this will always keep `SpeedControllerGroup.instances == 0`.

Reply:
Do I have to do `SpeedControllerGroup.instances += 1`

Reply:
Yup. Same with the other ones.